modules/gapi/misc/python/test/test_gapi_streaming.py

```diff
@@ -350,6 +349,67 @@ def test_compile_streaming_meta_and_args(self):
                     cv.gapi.compile_args(cv.gapi.streaming.queue_capacity(1)))
 
 
+        def test_gstreamer_source(self):
+            pipeline = """videotestsrc is-live=true pattern=colors num-buffers=10 !
+                          videorate ! videoscale ! video/x-raw,width=1920,height=1080,
+                          framerate=3/1 ! appsink"""
+
+            g_in = cv.GMat()
+            g_out = cv.gapi.copy(g_in)
+            c = cv.GComputation(cv.GIn(g_in), cv.GOut(g_out))
+
+            ccomp = c.compileStreaming()
+
+            # NB: Skip test in case gstreamer isn't available.
+            try:
+                source = cv.gapi.wip.make_gst_src(pipeline)
+            except cv.error as e:
+                raise unittest.SkipTest(str(e))
+
+            ccomp.setSource(cv.gin(source))
+            ccomp.start()
+
+            has_frame, output = ccomp.pull()
+            while has_frame:
+                self.assertTrue(output.size != 0)
+                has_frame, output = ccomp.pull()
+
+
+        def test_gstreamer_pipeline(self):
```

Comment:
What's the difference between: `test_gstreamer_source` and `test_gstreamer_pipeline` now sure that test names reflect it

I'd name test like this: `test_gst_two_inputs` or something like this

Reply:
OK, makes sense
It is kind of testing `GStreamerPipeline` class wrapped functionality (with testing multiple inputs, too) - that's why it was named after it